src/index.js

```diff
@@ -187,6 +188,17 @@ export default function PolymerRedux(store) {
 					});
 					return originalAction(...args);
 				};
+				// Copy props from the original action to the proxy.
+				// see https://github.com/tur-nr/polymer-redux/issues/98
+				Object.keys(originalAction).forEach(prop => {
+					action[prop] = originalAction[prop];
+				});
+
+				// Copy props from the original action to the proxy.
```

Comment:
We seem to a have a merge conflict, this is duplicated.

Reply:
I noticed. thx for the hint